src/development/ui/advanced/typography.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Flutter's fonts and typography
+description: Learn about Flutter's support for typography.
+---
+
+If you aren't familiar with [_typography_][], it essentially
+means how text is styled. [Variable fonts][] (also
+called OpenType fonts), allow you to control various
+aspects of text styling, such as font weight, width, slant,
+and so on.
```

Comment:
**issues:** The first sentence reads as uncertain. If the reader comes to this page, they either don't know about typography or don't know how it could be applied in Flutter. The description of typography needed some updating as did how variable type works.

Consider:
```suggestion
[_Typography_][] covers the style and appearance of type or fonts.
[Variable fonts][] allow one font to behave like multiple fonts.
You can control most all aspects of type for these fonts including
its weight, width, and variant. The OpenType file format would be
an example of a variable font.
```